Mu2eKinKal/src/BkgANNSHU.cc

```diff
@@ -23,21 +23,18 @@ namespace mu2e {
   WireHitState BkgANNSHU::wireHitState(WireHitState const& input, ClosestApproachData const& tpdata, DriftInfo const& dinfo, ComboHit const& chit) const {
     WireHitState whstate = input;
     if(input.updateable(StrawHitUpdaters::BkgANN)){
-      std::array<float,7> pars;
+      std::array<float,6> pars;
```

Comment:
For a future PR.  Is there a way to make this more robust - make the numeric literal a constexpr variable and then compare it to something you check against the ANN configuration that you read in?